src/main/java/com/spring/app/CountryLevel.java

```diff
@@ -1,5 +1,10 @@
 package com.spring.app;
 
+/**
+ * CountryLevel
+ * Contains a living level in country
+ * @author lyubov
```

Comment:
Bad english - live level, you've also forgotten an article before country

Reply:
changed